### DISCUSSION TITLE
Parcels with low-resolution GCM

### DISCUSSION THREAD

**Discussion by alexpohl** [Parcels-code/Parcels#1538] [Q&A]
Dear Erik, everyone,

I’d like to use a Lagrangian code to see how particules dropped at given locations spread within some given time duration in paleoclimate simulations conducted using a relatively low-resolution climate model.
This is FOAM, the Fast Ocean Atmosphere Model, the ocean resolution of which is 1.4° x 2.8° x 24 vertical levels (the surface level being 20 m thick).

Would the low resolution of FOAM lead to reasonable results in Parcels?

Thank you,
Best regards,
– Alexandre
__

Alexandre POHL
CNRS Researcher
Biogéosciences, Dijon, FRANCE
Personal webpage: https://alexpohl.github.io/

**Comment by erikvansebille** [comment]
Thanks for asking, @alexpohl. I don't know of many studies who have used Parcels on such coarse-resolution model output, so can't be certain about whether you get meaningful results or not. However, I do want to add two considerations/thoughts:

1. The temporal frequency of the output may be just as important as the spatial resolution. The longer the temporal averaging, the less variability will be captured. I assume you have monthly output? See e.g. [Weijer and Van Sebille (2014)](https://doi.org/10.1175/JCLI-D-12-00714.1) for an analysis of Agulhas leakage in coarse-res, monthly hydrodynamic flow data (although not with Parcels)
2. There is a difference between studies that aim to analyse the hydrodynamics _of a model_ using Lagrangian particles, and studies that aim to analyse a representation _of the real ocean_. In the first case, virtual particles can help you understand how water, heat and salt are transported within the model, and I reckon that this would be meaningful in coarse-res models too. But if you want to actually saw something about the real (paleo-)ocean, then there lack of eddies in low-res models might be a challenge

Perhaps @sruehs could add more thoughts, as she uses Parcels in climate model output for her MCSA project?

**Reply by alexpohl** [reply]
Thanks a lot Erik. In terms of temporal resolution, I would be able to generate the climatic output every month, week or even day, this is not a problem. Regarding the second point, my goal would be indeed to reconstruct the dispersion of particles in the paleo ('real') ocean – and I do understand that the FOAM-Parcels setup could then approach its limit. Based on this, I guess it would still be interesting and instructive to give this setup a try, while keeping in mind that it is important to interpret the results cautiously in terms of what they really mean. Thanks again. Any additional input from users very welcome indeed !
